lib/material_input/material_input.dart

```diff
@@ -61,6 +61,7 @@ const String materialInputErrorKey = 'material-input-error';
 /// - `multiple` -- Whether the user can enter multiple values, separated by
 ///   commas. This attribute only applies when type = "email", otherwise it is
 ///   ignored.
+/// - `role` -- The role attribute for the input element.
```

Comment:
Qualifying "role" as a "role attribute" doesn't say much :)  How about:

> `role` -- ARIA role for the input element.

Reply:
Leaving this for a future update.